Userland/Libraries/LibCore/System.cpp

```diff
@@ -36,6 +36,12 @@ ErrorOr<void> pledge(StringView promises, StringView execpromises)
     HANDLE_SYSCALL_RETURN_VALUE("pledge"sv, rc, {});
 }
 
+ErrorOr<void> revoke(StringView promises)
```

Comment:
This name is a bit generic... `pledge()` and `unveil()` as verbs are a bit unique in computing... but `revoke()` has meaning across many systems already. Is there a better name?

Reply:
- `repeal()`
- `retract()`
- `rescind()`
- `withdraw()`